pkg/scripts/uninstall.sh

```diff
@@ -94,7 +94,13 @@ regenerate_environment_variables() {
 
 destroy_infrastructure() {
   cd "${SCRIPT_PATH}/../../"
+  aws --region "${REGION}" eks update-kubeconfig --name "atlassian-dc-${ENVIRONMENT_NAME}-cluster"
   set +e
+  # Uninstall bamboo helm chart if is already installed - When we add new products we need to refactor this approach
+  if [ ! -z "$(helm list -n bamboo | grep bamboo)" ]; then
+    helm uninstall bamboo -n bamboo
```

Comment:
Just curious why we need to do this by helm CLI? terraform uses local machine CLI to add charts?

Reply:
reverted this